python/setup.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+# -*- coding:utf-8 -*-
+
+from setuptools import setup
+from setuptools import find_packages
+
+setup(
+    name="a6pluginproto",
+    version="0.2.0",
```

Comment:
Why start from 0.2?

Reply:
We have enough change to release the 0.2 after merging the https://github.com/api7/ext-plugin-proto/pull/14